lib/hardware/clocks_resets/iob_nco_sync/iob_nco_sync.py

```diff
@@ -0,0 +1,205 @@
+def setup(py_params_dict):
```

Comment:
this module is part of the nco , not a clock or reset

Reply:
Moved to iob_nco/hardware/modules in latest commit